wiki/en/Include-Backing-Up.md

```diff
@@ -1,15 +1,16 @@
 <!-- NOTE: This must apply to both Client and Server, and all operating systems -->
 
-Once installed and running, you may want to keep a copy of your settings. Having a backup is always a good idea, and settings files are not backwardly compatible between versions of Jamulus. So if you want to go back to the previous version, you will need to restore the settings you had. 
+Once installed and running, you may want to keep a copy of your settings. Having a backup is always a good idea, and settings files are not backwardly compatible between versions of Jamulus. So if you want to go back to the previous version, you will need to restore the settings you had.
 
-To find your settings file on Windows, look in the `%APPDATA%` folder. There will be one or more `.ini` files in a folder there named `Jamulus`.
+To find your settings file on Windows, launch the file manager and type `%APPDATA%` into the folder path box at the top. After that, search for the a folder named `Jamulus`. There will be one or more `.ini` files in this folder. Now back up Jamulus' settings by copying these files to a different location.
```

Comment:
I think this might be confusing on some setups. Probably more universal to say:

```suggestion
To find your settings file on Windows, type `%APPDATA%` into the search bar and look for a folder named `Jamulus`. There will be one or more `.ini` files in this folder. Now back up Jamulus' settings by copying these files to a different location.
```

Reply:
Also, "Now back up Jamulus' settings by copying these files to a different location." seems a bit redundant?

Reply:
In a para about _finding_ the settings, yes.

Reply:
Agree, but one could also say it's clarification what to do. But we can remove it.